### DISCUSSION TITLE
Is there any other place to ask for help?

### DISCUSSION THREAD

**Discussion by jairoamaral** [Stellarium/stellarium#3519] [General]
Stellarium is crashing all the time when controlling my AM5 telescope mount. I can't get directions to find a log file. And the telescope control plugin creator directs me to a Google Group where nobody answers.

**Comment by ilkant** [comment]
Put "Stlellarium crash all the time when controlling my AM5 telescope mount" to the Google search and look what people says to it. You can use also other (your preferred) search machines than Google.
